src/drivers/console/vc/vga.c

```diff
@@ -122,14 +122,15 @@ static void vc_diag_putc(const struct diag *diag, char ch) {
 	vterm_video_putc(&vc_vga_video.video, ch);
 }
 
-const struct diag_ops vc_video_diag_ops = {
+static const struct diag_ops vc_video_diag_ops = {
 	.init = vc_diag_init,
 	.putc = vc_diag_putc,
 };
-const struct vc_video_diag DIAG_IMPL_NAME(__EMBUILD_MOD__) = {
+
+static const struct vc_video_diag vc_video_diag = {
 	.diag = {
 		.ops = &vc_video_diag_ops,
 	},
 	.video = &vc_vga_video.video,
 };
-
+const struct diag *pdiag = &vc_video_diag.diag;
```

Comment:
Why not use DIAG_OPS_DEF?

Reply:
there is 'video' field in struct vc_video_diag except 'diag'

Reply:
But this structure is not used anywhere.

Reply:
Ops, thanks, fix

Reply:
But why don't use `DIAG_OPS_DEF`?